aptos-move/move-examples/cli-e2e-tests/common/sources/cli_e2e_tests.move

```diff
@@ -206,6 +206,20 @@ module addr::cli_e2e_tests {
         create_hero(account, description, gender, name, race, uri);
     }
 
+    public entry fun test_move_run(
```

Comment:
In the PR description you say `view` func but this is an entry function, just making sure you intended to use an entry function here 😃

Reply:
oops! I meant entry func. Good catch!